wolfcrypt/src/sha512.c

```diff
@@ -1426,10 +1456,19 @@ int wc_InitSha384_ex(wc_Sha384* sha384, void* heap, int devId)
     sha384->devId = devId;
     sha384->devCtx = NULL;
 #endif
+#if defined(WOLFSSL_USE_ESP32WROOM32_CRYPT_HASH_HW)
+        if (sha384->ctx.mode != ESP32_SHA_INIT) {
```

Comment:
Check indent here

Reply:
Ah yes, there was no reason for that indent. Fixed.